enterprise/metrics/src/main/java/org/neo4j/metrics/source/Neo4jMetricsBuilder.java

```diff
@@ -65,26 +68,26 @@
 
         TransactionCounters transactionCounters();
 
-        PageCacheMonitor pageCacheCounters();
+        PageCacheCounters pageCacheCounters();
 
         CheckPointerMonitor checkPointerMonitor();
 
         LogRotationMonitor logRotationMonitor();
 
-        @SuppressWarnings( "deprecation" )
-        IdGeneratorFactory idGeneratorFactory();
+        StoreEntityCounters entityCountStats();
 
-        DependencyResolver getDependencyResolver();
+        Supplier<ClusterMembers> clusterMembers();
```

Comment:
Because it is not initialized in construction, so we need to fetch it on `start` instead.